engine/src/main/java/io/camunda/zeebe/process/test/engine/GrpcResponseMapper.java

```diff
@@ -294,6 +300,27 @@ private GeneratedMessageV3 createModifyProcessInstanceResponse() {
     return ModifyProcessInstanceResponse.newBuilder().build();
   }
 
+  private GeneratedMessageV3 createMigrateProcessInstanceResponse() {
+    final var record = new ProcessInstanceMigrationRecord();
+    record.wrap(valueBufferView);
+
+    final var migrationPlanBuilder = MigrationPlan.newBuilder();
+    record.getMappingInstructions().stream()
+        .map(
+            instruction ->
+                MappingInstruction.newBuilder()
+                    .setSourceElementId(instruction.getSourceElementId())
+                    .setTargetElementId(instruction.getTargetElementId())
+                    .build())
+        .forEach(migrationPlanBuilder::addMappingInstructions);
+    final var migrationPlan = migrationPlanBuilder.build();
+
+    return MigrateProcessInstanceRequest.newBuilder()
```

Comment:
❌❓In the original response, we do not return any data. Is this intended to return data here for some purpose? @korthout

Reply:
Good point. I tried to be too fast, and missed that I used `*Request` rather than `*Response`. The request was asking for this data, so I added it without looking any further.

I've fixed it up to:
```java
private GeneratedMessageV3 createMigrateProcessInstanceResponse() {
  return ModifyProcessInstanceResponse.getDefaultInstance();
}
```